### DISCUSSION TITLE
No me pude aguantar de hacer el Pokedex :P

### DISCUSSION THREAD

**Discussion by sergiolopezloya** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#371] [Show and tell]
Ayer que dijo del Pokedex, simplemente no me pude aguantar, me encantan los retos de este tipo, ya lo terminé a como me parece que será, espero no tenga que cambiar algo:

https://pokedex.animeplay.us

Que opinan?
En el api no encontré nada para buscar, me hubiera gustado poder implementar una búsqueda porque las tablas son lentas.

**Comment by Ingrzs** [comment]
El Mc creo dijo que lo hariamos con otra consulta de la api de las habilidades

**Reply by sergiolopezloya** [reply]
Pues a ver cuándo ponga la actividad

**Comment by JuanSalvador73** [comment]
Quedó genial :D

**Reply by sergiolopezloya** [reply]
Gracias Juan

**Comment by Lfer1111** [comment]
se ve muy bien Sergio!! ..enseñanos =)

**Reply by sergiolopezloya** [reply]
Básicamente lo que vimos el viernes ;)

**Comment by Ricardo-g-garcia** [comment]
Te quedo muy bien, felicidades. Yo apenas intentando entender como consumir la API y como leer la documentacion, pero bueno algo saldra.

**Reply by sergiolopezloya** [reply]
Gracias, aunque ya vi los requerimientos y me faltan unos detalles, ya los haré durante la semana. Realmente no es muy complicado, con lo que vimos el viernes se puede lograr sin problema.

**Comment by DanyVeneno** [comment]
Tsss esta super !!!!!

**Comment by LuisFBecerrilJurado** [comment]
Te quedó super bien, me podrías asesorar a como limitar el campo de habilidades/movimientos a 2 o 4 opciones es que ando atorado en esa parte ya logre obtener todos los movimientos pero no hayo la forma de solo mostrar 4 (que es lo mas tradicional)

**Comment by Dfredude** [comment]
Podrias explicar como hiciste la animacion de las pokebolas?
